aptos-move/aptos-gas/src/instr.rs

```diff
@@ -15,176 +15,180 @@ crate::params::define_gas_parameters!(
+        [mul: InternalGas, "mul", 160],
+        [mod_: InternalGas, "mod", 160],
+        [div: InternalGas, "div", 160],
         // bitwise
-        [bit_or: InternalGas, "bit_or", 1],
-        [bit_and: InternalGas, "bit_and", 1],
-        [xor: InternalGas, "bit_xor", 1],
-        [shl: InternalGas, "bit_shl", 1],
-        [shr: InternalGas, "bit_shr", 1],
+        [bit_or: InternalGas, "bit_or", 160],
+        [bit_and: InternalGas, "bit_and", 160],
+        [xor: InternalGas, "bit_xor", 160],
+        [shl: InternalGas, "bit_shl", 160],
+        [shr: InternalGas, "bit_shr", 160],
         // boolean
-        [or: InternalGas, "or", 1],
-        [and: InternalGas, "and", 1],
-        [not: InternalGas, "not", 1],
+        [or: InternalGas, "or", 160],
+        [and: InternalGas, "and", 160],
+        [not: InternalGas, "not", 160],
         // comparison
-        [lt: InternalGas, "lt", 1],
-        [gt: InternalGas, "gt", 1],
-        [le: InternalGas, "le", 1],
-        [ge: InternalGas, "ge", 1],
-        [eq_base: InternalGas, "eq.base", 1],
+        [lt: InternalGas, "lt", 160],
+        [gt: InternalGas, "gt", 160],
+        [le: InternalGas, "le", 160],
+        [ge: InternalGas, "ge", 160],
+        [eq_base: InternalGas, "eq.base", 100],
         [
             eq_per_abs_val_unit: InternalGasPerAbstractValueUnit,
             "eq.per_abs_val_unit",
-            1
+            4
         ],
-        [neq_base: InternalGas, "neq.base", 1],
+        [neq_base: InternalGas, "neq.base", 100],
         [
             neq_per_abs_val_unit: InternalGasPerAbstractValueUnit,
             "neq.per_abs_val_unit",
-            1
+            4
         ],
         // global
         [
             imm_borrow_global_base: InternalGas,
             "imm_borrow_global.base",
-            10
+            500
         ],
         [
             imm_borrow_global_generic_base: InternalGas,
             "imm_borrow_global_generic.base",
-            10
+            500
         ],
         [
             mut_borrow_global_base: InternalGas,
             "mut_borrow_global.base",
-            10
+            500
         ],
         [
             mut_borrow_global_generic_base: InternalGas,
             "mut_borrow_global_generic.base",
-            10
+            500
         ],
-        [exists_base: InternalGas, "exists.base", 10],
-        [exists_generic_base: InternalGas, "exists_generic.base", 10],
-        [move_from_base: InternalGas, "move_from.base", 10],
+        [exists_base: InternalGas, "exists.base", 500],
+        [exists_generic_base: InternalGas, "exists_generic.base", 500],
+        [move_from_base: InternalGas, "move_from.base", 500],
         [
             move_from_generic_base: InternalGas,
             "move_from_generic.base",
-            10
+            500
         ],
-        [move_to_base: InternalGas, "move_to.base", 10],
+        [move_to_base: InternalGas, "move_to.base", 500],
```

Comment:
Does this have to be this expensive? We already charge for storage usage separately right?